### DISCUSSION TITLE
Coral pcie in Proxmox LXC

### DISCUSSION THREAD

**Discussion by Justahobby01** [blakeblackshear/frigate#894] [General]
Hello,

I am trying to run Frigate in an LXC on Proxmox using a mini-pcie coral. I can see the device in the lxc using the `lspci -nn | grep 089a` command. However, `ls /dev/apex_0` gives me `ls: cannot access '/dev/apex_0': No such file or directory`. 

As best as I can tell, I need to something in the lxc config like
`lxc.mount.entry: /dev/apex_0 /dev/apex_0 none bind,optional,create=file`

However, I don't see /dev/apex_0 when I use that. Is there something else I'm missing? LXC config below:

`arch: amd64
features: nesting=1
hostname: portainer
memory: 4096
net0: name=eth0,bridge=vmbr0,hwaddr=00,ip=dhcp,type=veth
onboot: 1
ostype: debian
rootfs: local-lvm:vm-102-disk-0,size=25G
swap: 512
lxc.cgroup.devices.allow: a
lxc.cap.drop: 
lxc.mount.entry: /dev/apex_0 /dev/apex_0 none bind,optional,create=file`

**Comment by hagak** [comment]
You probably need to install the PCIe driver:
https://coral.ai/docs/m2/get-started/#2-install-the-pcie-driver-and-edge-tpu-runtime

Not familiar with Proxmox so not sure if you can easily add that driver to it or not.

**Reply by Justahobby01** [reply]
Thanks for the response. I finally figured out I had an extra /

`lxc.mount.entry: /dev/apex_0 /dev/apex_0 none.... `

Should have been

`lxc.mount.entry:` /dev/apex_0 dev/apex_0 none... `
